src/libs/ajax.js

```diff
@@ -212,7 +216,7 @@ const User = signal => ({
             { id: 360012782111, value: email }
           ],
           comment: {
-            body: `${description}\n\n------------------\nSubmitted from: ${currUrl}`,
+            body: `${description}\n\n------------------\n${(emailAgreed === undefined) ? `Submitted from: ${currUrl}` : ``}`,
```

Comment:
`emailAgreed === undefined` is true when the request came from the contact us form, in which case having the "submitted from" location is useful. Otherwise, the request was from the return to FC modal, where the location doesn't matter.

Reply:
Even if it's not particularly relevant, I wonder if we should include it anyway just so we don't have to have special logic here. Is there a strong reason to exclude it?